index.html

```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                     <span class="error" id="error_message"></span>
                   </div>
                   <div class="form_group">
-                    <div class="row">
+                    <div class="row" style="margin: 0 auto; width: 656px; text-align: center;" >
                       <span id="success" class="col-xs-4"></span>
-                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 btn btn-lg send-button center-block" >
+                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 col-xs-pull-5 col-sm-pull-0 col-sm-push-0 btn btn-lg send-button center-block" >
```

Comment:
Get rid of all the `col` class here after you get the success message in a separate row.